.github/workflows/sdk_setup.yml

```
@@ -0,0 +1,30 @@
+- name: Setup Android SDK Tools
+  uses: android-actions/setup-android@v3.2.2
+
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 3: did not find expected '-' indicator

(syntax-check)